Jenkinsfile

```diff
@@ -1,18 +1,6 @@
 #!groovy
 
-import java.util.Collections
-
-// Valid Jenkins versions for test
-def testJenkinsVersions = [ '2.204.1', '2.204.6', '2.222.1', '2.222.4', '2.235', '2.240' ]
-Collections.shuffle(testJenkinsVersions)
-
-// Test plugin compatibility to subset of Jenkins versions
-subsetConfiguration = [ [ jdk: '8',  platform: 'windows', jenkins: testJenkinsVersions[0], javaLevel: '8' ],
```

Comment:
@MarkEWaite Sorry, but I am not able to understand what is wrong with the current config in the Jenkinsfile?